### ISSUE TITLE
[modules] [ini_manage] Provide a way to not have spaces between name separator and value

### ISSUE THREAD

**Issue by arthurlogilab** [saltstack/salt#38996]
### Description of Issue/Question

Feature requestion on the ini_manage module/state : some ini files require the syntax to be without spaces between key and value (for example etckeeper) : 

```
VCS="hg"
```
is correct
```
VCS = "hg"
```
is incorrect. 

The change concerns https://github.com/saltstack/salt/blob/develop/salt/modules/ini_manage.py#L328-L331 (I have a custom version in my ```_modules``` for now) 

### Versions Report

```
# salt --versions-report
Salt Version:
           Salt: 2016.3.4
 
Dependency Versions:
           cffi: 0.8.6
       cherrypy: Not Installed
       dateutil: 2.2
          gitdb: 0.5.4
      gitpython: 0.3.2 RC1
          ioflo: Not Installed
         Jinja2: 2.7.3
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: 1.0.0
   msgpack-pure: Not Installed
 msgpack-python: 0.4.2
   mysql-python: 1.2.3
      pycparser: 2.10
       pycrypto: 2.6.1
         pygit2: Not Installed
         Python: 2.7.9 (default, Jun 29 2016, 13:08:31)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 14.4.0
           RAET: Not Installed
          smmap: 0.8.2
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5
 
System Versions:
           dist: debian 8.6 
        machine: x86_64
        release: 3.16.0-4-amd64
         system: Linux
        version: debian 8.6 

```
cc @Akilesh1597

**Comment by gtmanfred** [comment]
I am marking this as a feature request.

If you wanted to submit a pull request to add this functionality, we would greatly appreciate it!

Thanks,
Daniel

**Comment by shallot** [comment]
This looks like a duplicate of https://github.com/saltstack/salt/issues/33669

**Comment by lorengordon** [comment]
Can probably be closed, as #31209 was merged. @gtmanfred?

**Comment by gtmanfred** [comment]
Yeah, looks like it.
